BeeSwift/SignInViewController.swift

```diff
@@ -25,6 +26,26 @@ class SignInViewController : UIViewController, UITextFieldDelegate {
     var backToSignUpButton = BSButton()
     var signInButton = BSButton()
     var divider = UIView()
+    let resetPasswordButton: BSButton = {
+        let button = BSButton()
+        button.setTitle("Forgot password?", for: .normal)
+        button.titleLabel?.font = UIFont.beeminder.defaultFontHeavy
+        if #available(iOS 13.0, *) {
```

Comment:
I probably would have at least grouped these `if ios13 else` together, setting both button title and background color in the same block.